bootstrap-salt.sh

```diff
@@ -6150,9 +6150,6 @@ config_salt() {
         exit 0
     fi
 
-    # Create default logs directory if not exists
-    mkdir -p /var/log/salt
-
```

Comment:
Actually these lines were added by me, but it looks like I had some other issue with directories and attempted to workaround it here. As for now I can't recall any details.

Nevertheless, I've tested the script without making logs dir, and it works just fine. All needed directories and log files being successfully created during packages installation and compilation from `git` sources.